example/src/main/java/org/secondbase/example/main/HelloSecondBase.java

```diff
@@ -15,8 +15,11 @@ private void printVariable() {
     }
 
     public static void main(final String[] args) throws SecondBaseException {
+        final String[] realArgs = new String[] {
+                "--service", "test"
+        };
         final HelloSecondBase helloSecondBase = new HelloSecondBase();
-        new SecondBase(args, new Flags().loadOpts(helloSecondBase));
+        new SecondBase(realArgs, new Flags().loadOpts(helloSecondBase));
```

Comment:
I explicitly added the variable variable now. I'm not sure what else to do. I agree it's not perfect that a variable is used inside a module, one that is mandatory. But there's no way around that with the scanning of flags from modules.